test/main_test.go

```diff
@@ -133,31 +133,35 @@ func configureTestOutput() {
 		panic(fmt.Errorf("Error creating junit file: %v", err))
 	}
 
-	opt.Output = io.MultiWriter(opt.Output, mainLogFile)
+	godogOpts.Output = io.MultiWriter(godogOpts.Output, mainLogFile)
+}
+
+func InitializeTestSuite(ctx *godog.TestSuiteContext) {
+	// Nothing done here for now
 }
 
-func FeatureContext(s *godog.Suite) error {
+func InitializeScenario(ctx *godog.ScenarioContext) {
 	// Create kube client
 	if err := framework.InitKubeClient(); err != nil {
```

Comment:
Would it have sense to move InitKubeClient and CheckSetup to InitializeTestSuite ? IMHO those are steps which need to be done just once.

Reply:
make sense yep. gonna move them